src/EnergyPlus/AirflowNetwork/include/AirflowNetwork/Elements.hpp

```diff
@@ -96,53 +96,17 @@ namespace AirflowNetwork {
 
     enum class DuctLineType
     {
-        Invalid = -1,
+        Unknown = 0,
```

Comment:
Ummm, why `Unknown = 0` as opposed to `Invalid = -1` which is what we are doing everywhere else?

Reply:
See other longer comment, I'm trying to get all these to the point where they're not read in or ever possibly invalid.